BP/common/casus_belli_types/00_religious_war.txt

```diff
@@ -0,0 +1,3717 @@
+				add_piety_experience = {
+					add = religious_cb_piety_gain_county
+					if = {
+						limit = {
+							scope:attacker.faith = {
+								has_doctrine_parameter = bonus_holy_war_piety_active
+							}
+						}
+						multiply = 1.5
+					}
+				}
+			}
+		}
+
+		scope:attacker.faith = {
+			change_fervor = -1
+		}
+		scope:defender.faith = {
+			change_fervor = 3
+		}
+
+		# Truce
+		add_truce_attacker_victory_effect = yes
+
+	}
+
+	on_white_peace_desc = {
+		desc = religious_war_white_peace_desc
+	}
+
+	on_white_peace = {
+		scope:attacker = { show_pow_release_message_effect = yes }
+
+		# Prestige for the attacker's war allies
+		add_from_contribution_attackers = {
+			prestige = religious_cb_ally_prestige_county
+			opinion = {
+				modifier = contributed_in_war
+			}
+		}
+
+		# Prestige for the defender's war allies
+		add_from_contribution_defenders = {
+			prestige = religious_cb_ally_prestige_county
+			opinion = {
+				modifier = contributed_in_war
+			}
+		}
+
+		# Truce
+		add_truce_white_peace_effect = yes
+
+		scope:attacker = {
+			add_piety = religious_cb_piety_white_peace
+		}
+	}
+
+	on_defeat_desc = {
+		desc = religious_war_defeat_desc
+	}
+
+	on_defeat = {
+		scope:attacker = { show_pow_release_message_effect = yes }
+
+		# Prestige for the attacker's war allies
+		add_from_contribution_attackers = {
+			prestige = religious_cb_ally_prestige_county
+			opinion = {
+				modifier = contributed_in_war
+			}
+		}
+		scope:attacker = {
+			every_vassal = {
+				limit = { faith = scope:attacker.faith }
+				custom = all_attackers_vassals_same_faith
+				add_opinion = {
+					modifier = liege_lost_declared_religious_war
+					target = scope:attacker
+				}
+			}
+		}
+
+		# Prestige for the defender's war allies
+		add_from_contribution_defenders = {
+			prestige = religious_cb_ally_prestige_county
+			opinion = {
+				modifier = contributed_in_war
+			}
+		}
+
+		# piety change
+		every_in_list = {
+			list = target_titles
+
+			scope:defender = {
+				add_piety = {
+					add = religious_cb_piety_gain_county
+					if = {
+						limit = {
+							scope:defender.faith = {
+								has_doctrine_parameter = bonus_holy_war_piety_active
+							}
+						}
+						multiply = 1.5
+					}
+				}
+			}
+
+			scope:attacker = {
+				pay_short_term_income = {
+					years = 2
+					target = scope:defender
+				}
+			}
+		}
+
+		# Truce
+		add_truce_attacker_defeat_effect = yes
+
+		scope:attacker = {
+			save_temporary_scope_as = loser
+		}
+		on_lost_aggression_war_discontent_loss = yes
+	}
+
+	on_primary_attacker_death = inherit
+	on_primary_defender_death = inherit
+
+	transfer_behavior = transfer
+
+	attacker_allies_inherit = yes
+	defender_allies_inherit = yes
+
+	war_name = "RELIGIOUS_WAR_NAME"
+	cb_name = "HOLY_WAR_COUNTY_NAME"
+
+	is_holy_war = yes
+
+	interface_priority = 80
+
+	use_de_jure_wargoal_only = yes
+
+	attacker_wargoal_percentage = 0.8
+
+	max_ai_diplo_distance_to_title = 500
+	min_ai_score = 100
+}
+
+religious_war = {
```

Comment:
Holy War should be disabled like in CK2, but what is the point of the changes you made in this file?

Reply:
It was; I noted in my institution proposal to gate holy wars by innovations:

* Early Migration Era: Doctrine and Dogma -> County Holy War
* Late Migration Era: Spiritual Canon -> Duchy Holy War
* Late Migration Era: Divine Mandate (Central Semitic Cultures) -> All Holy Wars
* Early Medevial Era: Established Faith -> Kingdom Holy War

Alternatively, we can completely block those with Established Faith, but I thought a progressive increase in scale seemed more logical. Similarly, the Arab cultures get to unlock all of the Holy Wars earlier to help model the Caliphates early expansion.

Reply:
The progressive scaling makes sense but I think it should be delayed in time, so County Holy War happens much later than the Early Migrational Era.

Reply:
County Holy Wars in the Late Migration, Duchy/Kingdom Holy Wars in the Early Medevial Era then, with Central Semetics and Muslim always having access to all holy wars?

Reply:
Doesn't the Rise of Islam happen by late Late Migration? I would disagree with Central Semetics always having access to all holy wars, to me that feels like railroading.

Reply:
Removed divine mandate and instead islam religion always has access to all holy wars. Furthermore, county holy war is now only available in late migration era and duchy/kingdom holy wars in early migration era.